src/Components/Facility/ConsultationForm.tsx

```diff
@@ -599,35 +600,17 @@ export const ConsultationForm = (props: any) => {
         form: {
           ...state.form,
           [e.target.name]: e.target.value,
-          // admitted: e.target.value === "A" ? "true" : "false",
         },
       });
   };
 
-  const handleSymptomChange = (e: any, child?: any) => {
-    const form = { ...state.form };
-    const { value } = e?.target;
-    const otherSymptoms = value.filter((i: number) => i !== 1);
-    // prevent user from selecting asymptomatic along with other options
-    form.symptoms =
-      child?.props?.value === 1
-        ? otherSymptoms.length
-          ? [1]
-          : value
-        : otherSymptoms;
-    form.hasSymptom = !!form.symptoms.filter((i: number) => i !== 1).length;
-    form.otherSymptom = !!form.symptoms.filter((i: number) => i === 9).length;
-    dispatch({ type: "set_form", form });
+  const handleValueChange = (value: any, field: string) => {
+    dispatch({
+      type: "set_form",
+      form: { ...state.form, [field]: value },
+    });
   };
 
-  // ------------- DEPRECATED -------------
-  // const handleDateChange = (date: any, key: string) => {
-  //   if (moment(date).isValid()) {
-  //     const form = { ...state.form };
-  //     form[key] = date;
-  //     dispatch({ type: "set_form", form });
-  //   }
-
```

Comment:
@cp-Coder 
- [ ] when we select asymptomatic, it should unselect all the already selected symptoms.
- [ ] when we select any other symptom, it should ensure that the asymptomatic is not selected.